conda-recipes/llvmlite/bld.bat

```diff
@@ -1,12 +1,22 @@
 
 @rem Let CMake know about the LLVM install path, for find_package()
 set CMAKE_PREFIX_PATH=%LIBRARY_PREFIX%
+
+@rem VS2019 uses a different naming convention for platforms than older version
 if "%ARCH%"=="32" (
-    set CMAKE_GENERATOR_ARCH=
+    @rem VS2017:
+    @rem set CMAKE_GENERATOR_ARCH=
+    set CMAKE_GENERATOR_ARCH=Win32
 ) else (
-    set CMAKE_GENERATOR_ARCH=Win64
+    @rem VS2017
+    @rem set CMAKE_GENERATOR_ARCH=Win64
+    set CMAKE_GENERATOR_ARCH=x64
 )
-set CMAKE_GENERATOR=Visual Studio 15 2017 %CMAKE_GENERATOR_ARCH%
+@rem for older VS:
+@rem set CMAKE_GENERATOR=Visual Studio 15 2017
+@rem do not set CMAKE_GENERATOR_TOOLKIT
+set CMAKE_GENERATOR=Visual Studio 16 2019
+set CMAKE_GENERATOR_TOOLKIT=v141
```

Comment:
Does this now mean llvmlite cannot be built in a conda-recipe with VS 2017?

Reply:
build.py will always try VS2019 toolkit v141 and then VS2017 if the generator select by the environment variables fails.  (See generators.extend() line later in patch.)

Reply:
Found it, thanks.